Makefile

```diff
@@ -84,7 +84,7 @@ clean:
 	rm -f $(install_dir)/firrtl.jar
 	rm -f $(install_dir)/firrtl
 	rm -f $(install_dir)/firrtl-stanza
-	"$(sbt)" "clean"
+	$(SBT) "clean"
```

Comment:
I just remembered why I got rid of the quotes, as @ucbjrl points out, I couldn't do:

SBT = java -Xmx8G -Xss512M -jar /path/to/sbt-launch.jar